src/styles/products.css

```diff
@@ -0,0 +1,47 @@
+.sidebar-container{
+    position: fixed;
+    left: 0%;
+    margin-top: 2.5rem;
+    background-color: var(--lynx-white-color);
+    height: 100vh;
+    width: 17rem;
+    padding: 3rem 1.5rem;
+    overflow-y: auto;
+}
+
+.category,.rating-label,.sort-label{
+    padding: 5px 0;
+}
+
+.container-box{
+    margin-bottom: 1.5rem;
+}
+
+.clear-filter:hover{
+    cursor: pointer;
+}
+
+.price-range{
+    width: 100%;
+}
+
+.product-container{
+ margin-left: 20rem;
+ padding: 4rem;
+}
+
+.product-img{
+    height: 70%;
+}
+
+.card{
+    background-color: white !important
+    
+}

```

Comment:
why are you using important here maybe try increasing specificity  by giving a addition card name

Reply:
Done [5b69145](https://github.com/Manpreet-hub/ecommerce-react/pull/2/commits/5b691456c4039caf5486bd9dd503c271e6193c2d)